site/gulpfile.js

```diff
@@ -1,15 +1,18 @@
 var gulp = require('gulp');
 var elixir = require('laravel-elixir');
+var gutils = require('gulp-util');
```

Comment:
`gulp-util` is a dependency of Laravel Elixir so it is available for requiring in the gulpfile. I was in two minds whether to add it to the jigsaw packages.json but elected not to in the end.